resources/assets/js/app.js

```diff
@@ -174,6 +174,14 @@ import axios from 'axios';
 import VueI18n from 'vue-i18n';
 Vue.use(VueI18n);
 
+// Moments
+import moment from 'moment';
+Vue.filter('formatDate', function(value) {
+    if (value) {
+        return moment(String(value)).format('LL')
```

Comment:
Great, but it must support locale. Just add `moment.locale(lang);` on line 210 after `loadLanguageAsync(window.Laravel.locale, true).then((lang) =>`

Reply:
Good point. I was lazy. Thanks.

Reply:
I've added it and tested it, it seems to work. Can you double check?

Reply:
@asbiin waiting for your validation here!

Reply:
Seems good!